juniper_official/Subscriber-Service/pppoe-error-statistics.rule

```diff
@@ -0,0 +1,255 @@
+iceberg {
+    topic service.protocols {
+        description "Rules related to PPPoE statistics monitoring of a JUNOS Broadband Network Gateway (BNG)";
+        synopsis "BNG PPPoE error statistics monitoring rules";
+        rule pppoe-error-statistics {
+            synopsis "Monitors PPPoE error statistics";
+            description "Collects the PPPoE error periodically and notifies in case of anomalies";
+            sensor pppoe-error-statistics {
+                synopsis "subscriber-management open-config sensor definition";
+                description "subscriber-management open-config sensor to collect telemetry data from network device";
+                open-config {
+                    sensor-name /junos/system/subscriber-management/client-protocols/pppoe/statistics;
+                    frequency 60s;
+                }
+            }
+            field ac-error-received {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/ac-error-received;
+                }
+                type integer;
+                description "Number of ac-error-received";
+            }
+            field ac-error-sent {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/ac-error-sent;
+                }
+                type integer;
+                description "Number of ac-error-sent";
+            }
+            field generic-error-received {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/generic-error-received;
+                }
+                type integer;
+                description "Number of generic-error-received";
+            }
+            field generic-error-sent {
+                sensor pppoe-error-statistics {
+                    path /junos/system/subscriber-management/client-protocols/pppoe/statistics/generic-error-sent;
+                }
+                type integer;
+                description "Number of generic-error-sent";
+            }
+            field malformed-packets-rec {
```

Comment:
Can we rename this to malformed-packtes-received to be consistent with the other triggers that end with either "received"?

If we really need to abbreviate it, can we change this to malformed-pkts-rx or some such, not just here, but in all cases?

Reply:
Not able to put received because of field length limit. Modified to malformed-pkts-rx.